src/Test/test_qtcore_main.cpp

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <gtest/gtest.h>
+
+#include <QCoreApplication>
+#include <cstdio>
+#include <optional>
+
+namespace {
+// QCoreApplication objects needs to be on the stack or in static storage. Putting this variable
+// into the QCoreApplicationStarter class leads to a crash during destruction. I'm not sure what is
+// going on.
+std::optional<QCoreApplication> app;
+
+// QCoreApplicationStarter is a gtest event listener which takes are of delayed construction of the
```

Comment:
Sentence derailed.

Reply:
Oh yes. There is a `c` missing which makes it completely not understandable. Thanks. :sweat_smile:

Reply:
Fixed.